src/app/components/MediaLoader/utils/buildSetting.test.ts

```diff
@@ -305,15 +303,14 @@ describe('buildSettings', () => {
           statsObject: {
             clipPID: 'p01k6msm',
             destination: 'WS_NEWS_LANGUAGES',
-            producer: 'MUNDO',
-            episodePID: null,
+            producer: 'SERBIAN',
           },
           enableToucan: true,
           externalEmbedUrl:
-            'https://www.bbc.com/mundo/av-embeds/srbija-68707945',
+            '/ws/av-embeds/cps/serbian/lat/srbija-68707945/p01k6msp/sr-latn',
```

Comment:
Is it ok for this to be building a URL without a host name and that starts with `/ws/` for a syndication asset?

Reply:
Its a good question. We may need some extra logic like you had before to cater specifically for CPS assets, and potentially another condition specifically for AMP to _always_ return the `/ws/` path regardless of asset type.

Let me take a look into this a bit more.

Reply:
I've added in the logic in https://github.com/bbc/simorgh/pull/11940/commits/2803fc49204d68e4b4a5e21bafdc42542b182790 to hopefully cater for both CPS and Optimo platforms, as the URLs are different for each.